web/src/scripts/containers/ProjectNavigator.jsx

```diff
@@ -117,6 +117,13 @@ class ProjectNavigator extends Component {
     textEditorCompositeActions.updateDecoEntryRequire(requireText)
   }
 
+  handleDrop = (sourceNode, targetNode) => {
+    this.props.fileTreeCompositeActions.renameFile(
```

Comment:
We probably want to call either `renameFile` or `renameDir` depending on what kind of `sourceNode` we're handling